hazelcast/src/main/java/com/hazelcast/map/impl/client/MapGetRequest.java

```diff
@@ -126,9 +126,6 @@ public String getDistributedObjectName() {
 
     @Override
     public String getMethodName() {
```

Comment:
yes this could lead to backwards incompatibility problems but we don't have a customer using this feature afaik (need to check). we've introduced this interceptor for a particular prospect who didn't purchase us at the end. 
I'm in favor of removing all those async method interceptors, if no one is using it.
the sooner we get rid of them the better